src/common/routes/config.ts

```diff
@@ -12,15 +12,15 @@ import { pdbRoute } from "./pod-disruption-budgets";
 import { resourceQuotaRoute } from "./resource-quotas";
 import { secretsRoute } from "./secrets";
 
-export const configRoute: RouteProps = {
+export const configRoute: RouterProps = {
   path: [
-    configMapsRoute,
-    secretsRoute,
-    resourceQuotaRoute,
-    limitRangesRoute,
-    hpaRoute,
-    pdbRoute,
-  ].map(route => route.path.toString()),
+    configMapsRoute.path,
+    secretsRoute.path,
+    resourceQuotaRoute.path,
+    limitRangesRoute.path,
+    hpaRoute.path,
+    pdbRoute.path,
+  ],
```

Comment:
Why change this? And did you mean to change all such instances? (like `networkRoute`)